invenio_communities/views/decorators.py

```diff
@@ -12,17 +12,31 @@
 
 from flask import g
 
+from invenio_communities.communities.resources.serializer import (
+    UICommunityJSONSerializer,
+)
 from invenio_communities.proxies import current_communities
 
 
-def pass_community(f):
+def pass_community(serialize):
     """Fetch the community record."""
 
-    @wraps(f)
-    def view(**kwargs):
-        pid_value = kwargs["pid_value"]
-        community = current_communities.service.read(id_=pid_value, identity=g.identity)
-        kwargs["community"] = community
-        return f(**kwargs)
-
-    return view
+    def decorator(f):
+        @wraps(f)
+        def view(**kwargs):
+            pid_value = kwargs["pid_value"]
+            community = current_communities.service.read(
+                id_=pid_value, identity=g.identity
+            )
+            kwargs["community"] = community
```

Comment:
Yes, we need to pass the `community` too because we also need to check for the permissions, see [here an example](https://github.com/inveniosoftware/invenio-communities/blob/5f46f455ccac721489b33e88b2877d07ae85feaa/invenio_communities/views/communities.py#L98).